lib/pdk/util.rb

```diff
@@ -100,5 +100,43 @@ def module_root
       end
     end
     module_function :module_root
+
+    # Returns the targets' paths relative to the working directory
+    #
+    # @ return Array<String> The absolute or path to the target
```

Comment:
We need to add a rake task to generate YARD output https://github.com/lsegal/yard#generating-documentation